src/App.js

```diff
@@ -1,26 +1,33 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import UserInput from './UserInput/UserInput';
+import UserOutput from './UserOuput/UserOutput';
+
+class App extends React.Component {
+  state = {
+    username: 'Olyaaaa',
+  }
+
+  userInputHandler = (event) => {
+    if (event.key === "Enter") {
+      this.setState({
+        username: event.target.value,
+      })
+    }
+  }
+
+  render() {
+    const size = 2;
+    const outputElements = Array(size).fill( <UserOutput username={this.state.username}/>);
```

Comment:
Ususally all states and props are **destructed** in the beginning of `render()` function

```
render() {
  const { className} = this.props
  const { username } = this.state
...
```

Reply:
Oo, I get it